libsplinter/src/biome/rest_api/actix/user.rs

```diff
@@ -112,21 +126,27 @@ fn add_fetch_user_method(credentials_store: Arc<SplinterCredentialsStore>) -> Ha
     })
 }
 
+#[cfg(feature = "biome-key-management")]
 /// Defines a REST endpoint to edit a user's credentials in the database
 ///
 /// The payload should be in the JSON format:
 ///   {
 ///       "username": <existing username of the user>
 ///       "hashed_password": <hash of the user's existing password>
-///       "new_password": OPTIONAL <hash of the user's updated password>
+///       "new_password": <hash of the user's updated password>
+///       "display_name": <display name for key>
+///       "public_key": <public key of the user>
```

Comment:
Users can be associated with multiple keys, so this needs to be a list of updated key pairs for all keys associated with the user.

Reply:
Addressed to take in a list of new key information, as well as return the successfully added keys.